tests/lib/rules/forward-ref-uses-ref.js

```diff
@@ -0,0 +1,267 @@
+/**
+ * @fileoverview Ensure symmetric naming of setState hook value and setter variables
```

Comment:
also i suspect this is the wrong description :-)

Reply:
Somewhere along the line git messed something up here. I wrote a description initially, I promise :)

Reply:
Ah it's because this is the test file 🤦